### ISSUE TITLE
Menu width slightly wider and set size of rigidly.

### ISSUE THREAD

**Issue by pyroxar** [PlayOnLinux/POL-POM-5#936]
Menu width needs to be slightly wider (up to red line shown in the picture) and its size rigid.
In this issue menu has diferent sizes in each tabs.
https://github.com/PlayOnLinux/POL-POM-5/issues/934
I think (but i don't know) that it is caused by size of menu's elements.
That's why you should change it.
![zrzut ekranu z 2017-06-28 12-25-32](https://user-images.githubusercontent.com/23406555/27633835-1da52414-5c01-11e7-94cc-55f0250dfeb4.png)
Long story short, the menu should be the same with as playonlinux logo.

**Comment by plata** [comment]
We cannot fix the width of the sidebar. If we would do this, the text of the buttons might be cropped.

However, I agree that it looks like the sidebar should have the same width as the logo. What would you think of:
- removing the logo completely
- moving the logo into the sidebar

**Comment by madoar** [comment]
Hmm, how about we bind the min-width of the sidebar to the width of the logo tab?

**Comment by plata** [comment]
I don't think that's a solution. What if the width of the logo is greater than the min-width of the sidebar?

**Comment by madoar** [comment]
I meant the other way round.
The sidebar will have as a minimum the width of the logo tab.

**Comment by plata** [comment]
That will not work if the sidebar has to be wider than the logo (e.g. because a text is longer). Also: do we really want to waste the space just to have different sidebars with the same width?

**Comment by madoar** [comment]
Why should it not work, if the sidebar needs to be wider?
We should bind the min-width property, not the max-width.
I'm not sure if we should call this wasting space. An application like POL 5 lives through its design, and I must agree with @pyroxar that it looks strange when you compare the width of the sidebar with the width of the logo tab, because you would expect them to have the same width.

**Comment by plata** [comment]
If a text is longer than the logo, the sidebar will be wider than the min-width. Therefore, sidebars will have different widths again.

**Comment by madoar** [comment]
And if we alternative bind the min width of the sidebar to the preferred width of the logo and the other way around? This should ensure that both components have the same width.

On another note:
What exactly speaks against fixed widths, with line breaks? This would require the translators to choose labels that look good in the application, but I think this is feasible.

**Comment by plata** [comment]
If you look at other sidebars, they usually do not break lines (e.g. in a file browser). Wouldn't it be easier to simply move the logo into the sidebar?

**Comment by madoar** [comment]
Can you do a PR?
If it looks good why not?

**Comment by plata** [comment]
![logo in sidebar](https://user-images.githubusercontent.com/3973260/27764095-35e768c4-5e91-11e7-8a9d-65c6ced06b79.png)

![logo in sidebar 2](https://user-images.githubusercontent.com/3973260/27764107-7eecc7ee-5e91-11e7-851c-6b20582e5b31.png)

**Comment by madoar** [comment]
I personally think a solution based on your second image looks best.
Can't be give the logo a blue background again, so that the header looks like a single element again?

**Comment by pyroxar** [comment]
I propose something like this:
![mzpropose](https://user-images.githubusercontent.com/23406555/27765752-d2f8ae4c-5eba-11e7-8a9e-deed0bc017df.png)

**Comment by plata** [comment]
@madoar I wouldn't want to do that. If we want the top menu to like like one, it should really be one. To be honest, I don't really see the point of having the logo in the application window at all. I cannot recall any other application which does this.
As the sidebar belongs to the tab and not the whole application, I prefer one of the last two images.

@qparis what do you think?
